openpype/tools/publisher/window.py

```diff
@@ -388,7 +396,9 @@ def resizeEvent(self, event):
     def closeEvent(self, event):
         self._window_is_visible = False
         self._uninstall_app_event_listener()
-        self.save_changes()
+        # TODO capture changes and ask user if wants to save changes on close
```

Comment:
Yes please! :)

Reply:
This is reason why the save button is enabled all the time...